core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -2729,7 +2920,381 @@ components:
+        limit:
+          description: The maximum number of transactions that will be returned.
+          type: integer
+    LtsStreamTransactionsBasicOutcomesResponse:
+      type: object
+      required:
+        - from_state_version
+        - count
+        - max_ledger_state_version
+        - committed_transaction_basic_outcomes
+      properties:
+        from_state_version:
+          type: integer
+          format: int64
+          minimum: 1
+          maximum: 100000000000000
+          description: |
+            An integer between `1` and `10^13`, giving the first (resultant) state version in the returned response (if any).
+            This matches the `from_state_version` in the request.
+        count:
+          type: integer
+          format: int32
+          minimum: 0
+          maximum: 10000
+          description: An integer between `0` and `10000`, giving the total count of transactions in the returned response
+        max_ledger_state_version:
+          type: integer
+          format: int64
+          minimum: 1
+          maximum: 100000000000000
+          description: An integer between `1` and `10^13`, giving the maximum state version currently committed on this node's ledger.
+        committed_transaction_basic_outcomes:
+          description: A committed transaction basic outcomes list starting from the `from_state_version` (inclusive).
+          type: array
+          items:
+            $ref: "#/components/schemas/LtsCommittedTransactionFungibleOutcome"
+    LtsCommittedTransactionStatus:
+      type: string
+      enum:
+        - Succeeded
+        - Failed
+      description: The status of the transaction
+    LtsCommittedTransactionFungibleOutcome:
+      type: object
+      required:
+        - state_version
+        - accumulator_hash
+        - status
+        - fungible_outcome
+      properties:
+        state_version:
+          type: integer
+          format: int64
+          minimum: 1
+          maximum: 100000000000000
+          description: An integer between `1` and `10^13`, giving the resultant state version after the transaction has been committed
+        accumulator_hash:
+          type: string
+          minLength: 64
+          maxLength: 64
+          description: |
+            The hex-encoded transaction accumulator hash. This hash captures the order of all transactions on ledger.
+            This hash is `ACC_{N+1} = Blake2b-256(CONCAT(ACC_N, LEDGER_HASH_{N}))`, starting with `ACC_0 = 000..000` the pre-genesis accumulator.
+        status:
+          $ref: "#/components/schemas/LtsCommittedTransactionStatus"
+        fungible_outcome:
+          description: A list containing all account fungible balance updates
+          type: array
+          items:
+            $ref: "#/components/schemas/LtsAccountFungibleBalanceChanges"
+    LtsAccountFungibleBalanceChanges:
```

Comment:
I think this should actually be LtsEntityFungibleBalanceChanges and include all entities - not just accounts.

As exchanges like sanity checking things like total balance change = 0 (less fees)